test/quantization/test_quant_api.py

```diff
@@ -43,8 +43,8 @@
     TORCH_VERSION_AFTER_2_4,
 )
 from pathlib import Path
-from sentencepiece import SentencePieceProcessor
-from model import Transformer, prepare_inputs_for_model
+from torchao.models.llama.tokenizer import get_tokenizer
+from torchao.models.llama.model import Transformer, prepare_inputs_for_model
```

Comment:
should we put these under `torchao._models` for now so people don't start using our models? @msaroufim  @cpuhrsch

Reply:
Yeah that sounds good

Reply:
Would torchtune users not be able to use them? :(

torchtune users can't currently [use ](https://pytorch.org/torchtune/stable/tutorials/llama3.html#faster-generation-via-quantization)torchao's GPTQ with a (fine-tuned) LLaMA3 because of [tokenization issues mentioned here](https://github.com/pytorch/ao/pull/276/files#r1619492654).

Reply:
i'll edit GPTQ to get tiktoken to work in the next PR, this one is mostly about llama2 functionality.

Reply:
Thanks in advance! :)

Reply:
tiktoken works with gptq but something is busted as far as llama3 eval goes, not sure what the issue is, the accuracy is just really bad